config/config.go

```diff
@@ -3,12 +3,13 @@ package config
 import "sync"
 
 const DefaultConfigTemplate = `
-FIRMWARE_ESP32={{ .FirmwareESP32 }}
-FIRMWARE_ESP32C3={{ .FirmwareESP32C3 }}
-PLANETMINT_ACTOR={{ .PlanetmintActor }}
-PLANETMINT_CHAIN_ID={{ .PlanetmintChainID }}
-SERVICE_BIND={{ .ServiceBind }}
-SERVICE_PORT={{ .ServicePort }}
+FIRMWARE_ESP32="{{ .FirmwareESP32 }}"
+FIRMWARE_ESP32C3="{{ .FirmwareESP32C3 }}"
+PLANETMINT_ACTOR="{{ .PlanetmintActor }}"
+PLANETMINT_CHAIN_ID="{{ .PlanetmintChainID }}"
+SERVICE_BIND="{{ .ServiceBind }}"
+SERVICE_PORT="{{ .ServicePort }}"
```

Comment:
Does this really work? ServicePort is an int, so there should be no quotes: `SERVICE_PORT={{ .ServicePort }}`